sdks/go/test/run_validatesrunner_tests.sh

```diff
@@ -70,7 +70,8 @@
 #        jar from the appropriate gradle module, which may not succeed.
 
 set -e
-set -v
+trap '! [[ "$BASH_COMMAND" =~ ^(echo|read|if|ARGS|shift|SOCKET_SCRIPT|\[\[) ]] && \
```

Comment:
Minor nit: A comment explaining what this line is doing would be useful, since it's pretty incomprehensible to the average reader.

(It just prints all commands executed to stdout except for the lines starting with one of those commands specified up top, right? I'm guessing the "eval echo" bit is some safety checks?)